docs/sources/tutorials/get-started.md

```diff
@@ -1,34 +1,49 @@
 ---
 canonical: https://grafana.com/docs/alloy/latest/tutorials/get-started/
-description: Getting started with the tutorials
-title: Get started
+description: Getting started with Alloy
+title: Get started with Alloy
 weight: 10
 ---
 
 ## Get started with {{% param "PRODUCT_NAME" %}}
 
-This set of tutorials contains a collection of examples that build on each other to demonstrate how to configure and use [{{< param "PRODUCT_NAME" >}}][alloy].
-To follow these tutorials, you need to have a basic understanding of what {{< param "PRODUCT_NAME" >}} is and telemetry collection in general.
-You should also be familiar with with Prometheus and PromQL, Loki and LogQL, and basic Grafana navigation.
-You don't need to know about the {{< param "PRODUCT_NAME" >}} [configuration syntax][configuration] concepts.
+This tutorial will walk you through configuring Alloy to collect logs from your local machine and report logs to Loki running in the local Grafana Stack. This process will enable you to query and visualize the logs sent to Loki using the Grafana dashboard.
+
+To follow this tutorial, you must have a basic understanding of Alloy and telemetry collection in general. You should also be familiar with Prometheus and PromQL, Loki and LogQL, and basic Grafana navigation. You don't need to know about the 
+{{< param "PRODUCT_NAME" >}} [configuration syntax][configuration] concepts.
```

Comment:
I think it makes more sense to call it a stack; that's what it is; also I'd rather not get into enumerating everything that's in it, because that will get long (there are several components, and most aren't relevant in this short context).